packages/ui-contacts/src/customers/components/common/DetailInfo.tsx

```
@@ -90,6 +86,71 @@ class DetailInfo extends React.PureComponent<Props> {
+                <td>{__("mobile")}</td>
+              </tr>
+            )}
+            {phones[2] && (
+              <tr>
+                <td>{phones[2].phone || "-"}</td>
+                <td>{__("work")}</td>
+              </tr>
+            )} */}
+            {
+              phones.map(phone => (
+                <tr>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)